src/main/java/frc/robot/motors/MotorManager.java

```diff
@@ -0,0 +1,55 @@
+package frc.robot.motors;
+
+import java.util.LinkedList;
+import java.util.List;
+
+import org.littletonrobotics.junction.Logger;
+
+import frc.robot.Robot;
+
+/**
+ * The motor manager that facilitates pooling and logging motor outputs.
+ */
+public class MotorManager {
+
+	/** A global list containing all logged motors. */
+	private static List<LoggedMotor> motorList = new LinkedList<>();
```

Comment:
Use an ArrayList, not a LinkedList. ArrayList is vastly more efficient in almost all situations.